packages/stylelint-config/package.json

```diff
@@ -6,11 +6,11 @@
   "license": "Apache-2.0",
   "main": "index.js",
   "dependencies": {
-    "stylelint-config-prettier": "^8.0.2",
-    "stylelint-config-standard": "^20.0.0"
+    "stylelint-config-prettier-scss": "^0.0.1",
+    "stylelint-config-standard-scss": "^3.0.0"
```

Comment:
So that it uses scss linting rules instead of just css linting rules. Take a look at https://stylelint.io/user-guide/get-started/#using-a-community-shared-config